File: Program's_Contributed_By_Contributors/Java_Programs/Data_structure/LinearSearch.java
class LinearSearch {
    public static void main(String[] args) {
        int[] arr = {1, 4, 2, 5, -1};
        int target = 2;
        boolean flag = false;
        for(int i = 0; i < arr.length; i++) {
            if(arr[i] == target) {
                System.out.println("Target found at index: " + i);
                flag = true;
                break;
            }
        }
        if(flag == false) System.out.println("Target is not present");
    }
}

File: C-Sharp/Searching_Algorithms/Linear_search.cs
using System;

public class LinearSearchExample
{
    public static int LinearSearch(int[] arr, int target)
    {
        for (int i = 0; i < arr.Length; i++)
        {
            if (arr[i] == target)
                return i; // Element found at index i
        }
        return -1; // Element not found
    }

    public static void Main()
    {
        int[] arr = { 10, 25, 5, 15, 30, 20, 35 };
        int target = 15;

        int result = LinearSearch(arr, target);

        if (result != -1)
            Console.WriteLine("Element found at index " + result);
        else
            Console.WriteLine("Element not found in the array.");
    }
}
